roles/eos-l3ls-evpn/templates/tenant-evpn-vxlan/leaf-vxlan-vtep-interface.j2

```diff
@@ -19,6 +19,6 @@
 ## {{ tenant }} ##
 {%     for vrf in tenants[tenant].vrfs | natural_sort %}
         {{ vrf }}:
-          vni: {{ vxlan_l3_vni_base + tenants[tenant].id *100 + tenants[tenant].vrfs[vrf].id }}
+          vni: {{ vxlan_vni_base * tenants[tenant].id + 5000 + tenants[tenant].vrfs[vrf].id  }}
```

Comment:
Instead of defining VNI base in settings and multiplication, shouldn't we just define a meaningful VNI format.
Format can be different, for example:
IP-VRF VNI:
``{{  '%04d'|format(tenants[tenant].vrfs[vrf].id) }}``
MAC-VRF:
``{{ '%04d'|format(tenants[tenant].id) }}{{ '%02d'|format(scope_number) }}{{ '%04d'|format(svi) }}``
Format can be completely different, but the idea is still to produce some meaningful VNI that can be adjusted based on design requirements.